cmd/argocd/commands/admin/backup.go

```diff
@@ -472,3 +500,17 @@ func updateTracking(bak, live *unstructured.Unstructured) {
 		}
 	}
 }
+
+// skip resource   if any of the specified label exists.
+func isSkipLabelMatches(bak *unstructured.Unstructured, skipResourcesWithLabels []string) bool {
```

Comment:
I'm not really a fan of the "is label present" filter. Seems at odds with the way most label filtering works. Is there any reason not to go with `label=value` filtering?

Reply:
Label value has not very much usage here when filtering the resource. There is not issue in adding value but usually key would be better way to use.  See Alex [comment](https://github.com/argoproj/argo-cd/pull/21694#discussion_r1932765490)
I can add kv if needed

Reply:
@crenshaw-dev  changes are done [here](https://github.com/argoproj/argo-cd/pull/21694/files#diff-d65db0513f73b640208f3057b98224e9f38372f4104b8a074c4828a5414dca0fR516)